scripts/UpdateTwimlApp.js

```diff
@@ -0,0 +1,110 @@
+"use strict";
+const https = require("node:https");
+const http = require("node:http");
+const process = require("node:process");
+const NGROK_URL = "http://localhost:4040/api/tunnels";
+const TWIML_APP_URL = (accountSid, appSid) =>
+  `https://api.twilio.com/2010-04-01/Accounts/${accountSid}/Applications/${appSid}.json`;
+const ACCOUNT_SID = process.env.ACCOUNT_SID;
+const TWIML_APP_SID = process.env.TWIML_APP_SID;
+const API_KEY_SID = process.env.API_KEY_SID;
+const API_KEY_SECRET = process.env.API_KEY_SECRET;
+
+function createJsonResponseHandler(resolve, reject) {
+  return (response) => {
+    const statusCode = response.statusCode;
+    const contentType = response.headers["content-type"];
+    if (statusCode !== 200) {
+      response.resume();
+      return reject(new Error(`status code: ${statusCode}`));
+    }
+    if (!/application\/json/.test(contentType)) {
+      response.resume();
+      return reject(new Error(`content type: ${contentType}`));
+    }
+    response.setEncoding("utf8");
+    const rawDataChunks = [];
+    response.on("data", (chunk) => {
+      rawDataChunks.push(chunk);
+    });
+    response.on("end", () => {
+      resolve(JSON.parse(rawDataChunks.join()));
+    });
+  };
+}
```

Comment:
Handling requests like this tend to be brittle as there are a lot of cases that needs to be handled. The happy path works but this is usually not encourage. A lot of libraries out there are already well tested and made for only handling requests. Why did we not use them (axios/request/superagent)? This file would be so much simpler.

Reply:
@mhuynh5757 was there a reason for this? Or should I change this request to axios?

Reply:
No need to change now if it adds significant amount of effort. But I let you make the call.

Reply:
I wrote it this way for personal use, I didn't want a package just for axios and this one script. Since we're already depending on axios, I'm in favor of using axios (it'll cut the script size down in half honestly).

Reply:
Understood. @kpchoy I'll let you make the call to rewrite this. It's probably not worth it if it's too much work.

Reply:
i'll create a ticket for the backlog since I am short on time for this ticket